dev-libs/libsodium/libsodium-1.0.16.ebuild

```diff
@@ -1,9 +1,9 @@
-# Copyright 1999-2017 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=6
 
-inherit eutils autotools
+inherit multilib-minimal eutils autotools
```

Comment:
Please sort inherits lexically. Also, I think you don't use `eutils` (anymore).

Reply:
Sure, will do. I thought, it was on purpose, as the original ebuild was reversed lexically ;-)